GenumKit/Parsers/FontsFileParser.swift

```diff
@@ -35,16 +35,18 @@ public func == (lhs: Font, rhs: Font) -> Bool {
 // MARK: CTFont
 
 extension CTFont {
-  static func parseFontInfo(fileURL: NSURL) -> Font? {
+  static func parseFontInfo(fileURL: NSURL) -> [Font]? {
```

Comment:
Now that this method returns an array and not a single font, maybe we could return a non-optional array instead (and return an empty array on any `guard` failure)?